docs/fidesops/docs/guides/configuration_reference.md

```diff
@@ -117,6 +117,8 @@ Please note: The configuration is case-sensitive, so the variables must be speci
 | `FIDESOPS__DEV_MODE` | False | If "True", the fidesops server will log error tracebacks, and log details of third party requests. This variable should always be set to "False" in production systems.|
 | `FIDESOPS__CONFIG_PATH` | None | If this variable is set to a path, that path will be used to load .toml files first. That is, any .toml files on this path will override any installed .toml files. |
 | `FIDESOPS__DATABASE__SQLALCHEMY_DATABASE_URI` | None | An optional override for the URI used for the database connection. |
+| `TESTING` | False | This variable does not need to be set - Pytest will set it to True when running unit tests, so we run against the test database. |
```

Comment:
I intentionally left this out of the docs originally as it's more of an implementation detail. I can see it's good for transparency to show it, might it confuse some users too?

Reply:
Your call, I can go either way.  Most users probably don't need this, but people digging in the code might wonder what test mode is.

Reply:
I'm okay with leaving this in for now, as I intend to refactor this page at some point, but I do think generally that anything targeted for testing or development should ideally live in the Development section - we just don't have a good spot for it yet.